### ISSUE TITLE
Bottom nav panel links: always show text

### ISSUE THREAD

**Issue by abettermap** [Language-Mapping/language-map#191]
Bottom navigation panel icons: always show text, not just for the active panel. _Note: this may result in very small text on mobile screens._

**Comment by abettermap** [comment]
@rperlin-ela good suggestion whoever made it. I took some liberties on style changes too. Some nuances:

The active one on desktop has slightly larger text (but not on mobile since not enough room). Here is desktop:

![image](https://user-images.githubusercontent.com/4974087/111537942-ae28f880-8731-11eb-87f3-54398086d64f.png)

I used a gradient background for active tab, let me know if it's too much but I think it's not excessive in that small space. And it was more exciting than the flat color.

We never actually had text on mobile, even for active! It fits though (as long as we never have longer words haha):

![image](https://user-images.githubusercontent.com/4974087/111538345-2e4f5e00-8732-11eb-9c45-a27493003cbe.png)
